odk1-src/collect-connect.rst

```diff
@@ -20,9 +20,9 @@ Configure server from QR code
   :alt: Configure via QR code
   :class: device-screen-vertical
 
-1. Position the QR code in the center of the camera field, under the red line. When the camera focuses on the code, it will beep and scan the code.
+2. Position the QR code in the center of the camera field, under the red line. When the camera focuses on the code, it will beep and scan the code.
 
-1. Collect will apply the settings from the code and go back to the landing screen.
```

Comment:
rst will correctly update the numbers if you use 1 everywhere. This is preferable to numbering manually (same in markdown).

Reply:
This wasn't working on this page for some reason which is why I changed it. Can try again.